draft-ietf-quic-multipath.md

```diff
@@ -863,9 +682,8 @@ Therefore, the packet number space for 1-RTT packets can be identified
 based on the Destination Connection ID in each packet.
 
 As soon as the negotiation of multipath support is completed,
-endpoints SHOULD use ACK_MP frames instead of ACK frames for
-acknowledgements of 1-RTT packets on path 0, as well as for 0-RTT packets
-that are acknowledged after the handshake concluded.
+endpoints SHOULD use ACK_MP frames instead of ACK frames also for 0-RTT packets
+that are acknowledged after the handshake concluded using the Packet Number Space Identifier 0.
```

Comment:
Maybe clearer?
```suggestion
endpoints SHOULD use ACK_MP frames instead of ACK frames to acknowledge application
data packets, including 0-RTT packets, received on path ID 0 after the handshake concluded.
```